vmdb/app/models/vm_or_template/operations/smart_state.rb

```diff
@@ -0,0 +1,9 @@
+class VmOrTemplate
+  module Operations
+    module SmartState
+      def validate_scan
+        validate_vm_control_powered_on
```

Comment:
Check does not seem there in the UI/action tasks/ path for scanning.  Will update that method.